README.md

```diff
@@ -72,6 +72,13 @@ You can also install directly from source:
 pip install "id-mava[reverb,jax,envs] @ git+https://github.com/instadeepai/mava.git"
 ```
 
+If you would like to develop features for Mava, the best way to do download all dependencies is:
```

Comment:
minor fix. 
"If you would like to develop features for Mava, please install locally from source as follows:"